app/controllers/ReviewReasonController.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import connectors.GoodsRecordConnector
+import controllers.actions._
+
+import javax.inject.Inject
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.ReviewReasonView
+
+import scala.concurrent.ExecutionContext
+
+class ReviewReasonController @Inject() (
+  override val messagesApi: MessagesApi,
+  identify: IdentifierAction,
+  getData: DataRetrievalAction,
+  requireData: DataRequiredAction,
+  val controllerComponents: MessagesControllerComponents,
+  view: ReviewReasonView,
+  goodsRecordConnector: GoodsRecordConnector
+)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
+    with I18nSupport {
+
+  def onPageLoad(recordId: String): Action[AnyContent] = (identify andThen getData andThen requireData).async {
+    implicit request =>
+      goodsRecordConnector
+        .getRecord(request.eori, recordId)
+        .map { record =>
+          record.reviewReason match {
```

Comment:
are we translating reviewReason somewhere

Reply:
Just spoken with Liandra and Ben, there's some debate on whether what the frontend gets in the goodsRecord is the longer review reason or shorter reason. For now, review as normal.

NO MERGING tho :)

Reply:
The shorter reason will be used on the frontend side, what's displayed to the user is defined in the messages file, however